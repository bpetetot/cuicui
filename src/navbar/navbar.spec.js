/* eslint-disable react/jsx-filename-extension */
/* eslint-env jest */
import snap from 'snap'

import Navbar from './navbar'

const snapshot = props => snap(Navbar)({ ...props })

describe('navbar', () => {
  it('should add logo', snapshot({ logo: 'hello world' }))
  it('should add menu', snapshot({ children: 'hello world' }))
  it('should add custom className', snapshot({ className: 'custom' }))
  it('should add custom style', snapshot({ style: { color: 'red' } }))
})
