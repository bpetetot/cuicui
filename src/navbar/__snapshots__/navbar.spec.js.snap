// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`navbar should add custom className 1`] = `
<header
  className="cc-navbar custom"
  style={undefined}
>
  <div
    className="cc-navbar--container"
  >
    <div
      className="cc-navbar--logo"
    />
    <nav
      className="cc-navbar--menu"
    />
  </div>
</header>
`;

exports[`navbar should add custom style 1`] = `
<header
  className="cc-navbar"
  style={
    Object {
      "color": "red",
    }
  }
>
  <div
    className="cc-navbar--container"
  >
    <div
      className="cc-navbar--logo"
    />
    <nav
      className="cc-navbar--menu"
    />
  </div>
</header>
`;

exports[`navbar should add logo 1`] = `
<header
  className="cc-navbar"
  style={undefined}
>
  <div
    className="cc-navbar--container"
  >
    <div
      className="cc-navbar--logo"
    >
      hello world
    </div>
    <nav
      className="cc-navbar--menu"
    />
  </div>
</header>
`;

exports[`navbar should add menu 1`] = `
<header
  className="cc-navbar"
  style={undefined}
>
  <div
    className="cc-navbar--container"
  >
    <div
      className="cc-navbar--logo"
    />
    <nav
      className="cc-navbar--menu"
    >
      hello world
    </nav>
  </div>
</header>
`;
